### PR TITLE
Add docs about directionality

### LINKED ISSUE #724: Document directionality
From 2.1. onwards we start to support directionality. We need to document this feature.

### PR DESCRIPTION
**Pull request by tobi-or-not-tobi**
closes #724

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -134,10 +134,12 @@ docs:
           - title: "Styling and Page Layout ➢"
             url: /styling-and-page-layout/
             subchildren:
-                - title: "Global Variables"
-                  url: /global-variables/
                 - title: "CSS Architecture"
                   url: /css-architecture/
+                - title: "Directionality"
```
Comment (tobi-or-not-tobi) [inline _data/navigation.yml:139]: I assume this is about an order of the sub pages. If so, I prefer to have it after the page layout docs.

File: _pages/dev/styling-and-page-layout/directionality.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 2.1 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The directionality feature provides support for bidirectional text and layout. You can configure Spartacus to use a left-to-right (LTR) orientation, or a right-to-left (RTL) orientation.
+
+Directionality is driven by language. Many languages are read from left to right, but well known examples, such as Arabic and Hebrew, are read from right to left.
```
Comment (tobi-or-not-tobi) [inline _pages/dev/styling-and-page-layout/directionality.md:13]: > but well known examples, such as Arabic and Hebrew, are read from right to left
this reads funny to me. Should it not be "but some languages, such as Arabic and Hebrew, are read from right to left"

File: _pages/dev/styling-and-page-layout/directionality.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+In Spartacus, the direction of the UI reflects the active language, so that directionality can work in a multi-directional experience. If your storefront contains both LTR and RTL languages, the active language is used to detect the direction automatically.
```
Comment (tobi-or-not-tobi) [inline _pages/dev/styling-and-page-layout/directionality.md:15]: "bi-directionality" is a common wording, I'd prefer to stick to it. Loads of frameworks made the bidi abbreviation out of it.

File: _pages/dev/styling-and-page-layout/directionality.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+## Breaking Changes
+
+The changes made to the DOM and CSS are considered breaking changes, so the directionality feature can only be used if you enable it with the 2.1 feature flag and the 2.1 CSS version.
```
Comment (tobi-or-not-tobi) [inline _pages/dev/styling-and-page-layout/directionality.md:19]: can we add a link to the css versioning page, give that it's also a new feature with 2.1?

File: _pages/dev/styling-and-page-layout/directionality.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    direction: {
+        default: DirectionMode.RTL,
+        ltrLanguages: ['en'],
+    },
+} as DirectionConfig),
+```
+
+For more information on configuration, see [Global Configuration in Spartacus]({{ site.baseurl }}{% link _pages/dev/global-configuration-in-spartacus.md %}).
+
+## Implementing Directionality
```
Comment (tobi-or-not-tobi) [inline _pages/dev/styling-and-page-layout/directionality.md:50]: I don't like that title. What about implementation details?

File: _pages/dev/styling-and-page-layout/styling-and-page-layout.md [hunk 6]
```diff
@@ -4,7 +4,8 @@ title: Styling and Page Layout
 
 This is a landing page for grouping together topics about styling and page layout. These topics include the following:
 
-- [Global Variables]({{ site.baseurl }}{% link _pages/dev/styling-and-page-layout/global-variables.md %})
 - [CSS Architecture]({{ site.baseurl }}{% link _pages/dev/styling-and-page-layout/css-architecture.md %})
+- [Directionality]({{ site.baseurl }}{% link _pages/dev/styling-and-page-layout/directionality.md %})
```
Comment (tobi-or-not-tobi) [inline _pages/dev/styling-and-page-layout/styling-and-page-layout.md:8]: same here, I'd move directionality below page layout. It's less important.

File: _data/navigation.yml (hunk 1 above)
Comment (gladius-mtl) [reply _data/navigation.yml:139]: Fixed 🙂

File: _pages/dev/styling-and-page-layout/styling-and-page-layout.md (hunk 6 above)
Comment (gladius-mtl) [reply _pages/dev/styling-and-page-layout/styling-and-page-layout.md:8]: Fixed 🙂

File: _pages/dev/styling-and-page-layout/directionality.md (hunk 2 above)
Comment (gladius-mtl) [reply _pages/dev/styling-and-page-layout/directionality.md:13]: Done 🙂

File: _pages/dev/styling-and-page-layout/directionality.md (hunk 3 above)
Comment (gladius-mtl) [reply _pages/dev/styling-and-page-layout/directionality.md:15]: Yes, "bidirectional" makes much more sense than "multi-directional", in this case 😄

File: _pages/dev/styling-and-page-layout/directionality.md (hunk 4 above)
Comment (gladius-mtl) [reply _pages/dev/styling-and-page-layout/directionality.md:19]: Done 👍

File: _pages/dev/styling-and-page-layout/directionality.md (hunk 5 above)
Comment (gladius-mtl) [reply _pages/dev/styling-and-page-layout/directionality.md:50]: 👍

**Pull request merged**